tests/src/Simple/ReadyToRunUnit/Program.cs

```diff
@@ -0,0 +1,570 @@
+            if (index > 0 && listOfInt[index] <= listOfInt[index - 1])
+            {
+                // The list should be monotonically increasing now
+                return false;
+            }
+        }
+        Console.WriteLine();
+        return listOfInt.Count == ItemCount;
+    }
+
+    private static bool ConstructListOfString()
+    {
+        List<string> listOfString = new List<string>();
+        return listOfString.Count == 0;
+    }
+
+    private static bool ManipulateListOfString()
+    {
+        List<string> listOfString = new List<string>();
+        const int ItemCount = 100;
+        for (int index = ItemCount; index > 0; index--)
+        {
+            listOfString.Add(index.ToString());
+        }
+        listOfString.Sort();
+        //listOfInt.Sort((a, b) => a.CompareTo(b));
+        for (int index = 0; index < listOfString.Count; index++)
+        {
+            Console.Write($@"{listOfString[index]} ");
+            if (index > 0 && listOfString[index].CompareTo(listOfString[index - 1]) <= 0)
+            {
+                // The list should be monotonically increasing now
+                return false;
+            }
+        }
+        Console.WriteLine();
+        return listOfString.Count == ItemCount;
+    }
+    //*/
+
+    private static bool EmptyArray()
+    {
+        int[] emptyIntArray = Array.Empty<int>();
+        Console.WriteLine("Successfully constructed Array.Empty<int>!");
+        return emptyIntArray.Length == 0;
+    }
+
+    private delegate char CharFilterDelegate(char inputChar);
+
+    private static bool CharFilterDelegateTest()
+    {
+        string transformedString = TransformStringUsingCharFilter(TextFileName, CharFilterUpperCase);
+        Console.WriteLine(transformedString);
+        return transformedString.Length == TextFileName.Length;
+    }
+
+    private static string TransformStringUsingCharFilter(string inputString, CharFilterDelegate charFilter)
+    {
+        StringBuilder outputBuilder = new StringBuilder(inputString.Length);
+        foreach (char c in inputString)
+        {
+            char filteredChar = charFilter(c);
+            if (filteredChar != '\0')
+            {
+                outputBuilder.Append(filteredChar);
+            }
+        }
+        return outputBuilder.ToString();
+    }
+
+    private static char CharFilterUpperCase(char c)
+    {
+        return Char.ToUpperInvariant(c);
+    }
+
+    //*
+    private static bool EnumerateEmptyArray()
+    {
+        foreach (int element in Array.Empty<int>())
+        {
+            Console.Error.WriteLine($@"Error: Array.Empty<int> has an element {element}!");
+            return false;
+        }
+        foreach (string element in Array.Empty<string>())
+        {
+            Console.Error.WriteLine($@"Error: Array.Empty<string> has an element {element}");
+            return false;
+        }
+        return true;
+    }
+    //*/
+
+    public static int Main()
+    {
+        /*
```

Comment:
Should this file be cleaned up at least a little? [Broken windows theory](https://en.wikipedia.org/wiki/Broken_windows_theory) says this file is only going to get worse. I've seen that happen far too many times in practice.

Reply:
Yes - I took it verbatim from Tomas' current branch. I'll remove the bringup check list. We can store it in a OneNote or something instead.

Reply:
Sorry about the excessive junk, of course feel free to remove anything.